update.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import os
+from datetime import datetime
+from jinja2 import Environment, FileSystemLoader
+from typing import NamedTuple
+from urllib.error import URLError
+from urllib.request import urlopen, Request
+
+JDK_URLS = {
+    (12, 0, 1): 'https://download.java.net/java/GA/jdk12.0.1/69cfe15208a647278a19ef0990eea691/12/GPL/openjdk-12.0.1_linux-x64_bin.tar.gz',
```

Comment:
shall the keys be of type `Version`, coz we look them up by https://github.com/crate/docker-crate/pull/156/files#diff-e541821577b38a3845c3adfc571d4baaR81 where the jdk version is a named tuple.